healpy/pixelfunc.py

```diff
@@ -131,6 +131,8 @@
     "npix2nside",
     "nside2order",
     "order2nside",
+    "order2npix",
+    "npix2order",
```

Comment:
can you also import them into `__init__.py`?

Reply:
Ah, sorry for that! I've imported them.